Add files via upload

breadsp opened the pull request:
Cleaned up white-space warnings from Codacy
Made all SQL queries prepared Statements
Removed the pre-validated DB code (was commented out, have saved in different version)

breadsp committed:
Add files via upload

Cleaned up white-space warnings from Codacy
Made all SQL queries prepared Statements
Removed the pre-validated DB code (was commented out, have saved in different version)

breadsp committed:
Add files via upload

All File (Submission) test cases have been manually checked
Cleaned up code to make compliant with Codacy suggestions

n2iw commented:
- line 120 calling enumerate function is not necessary
- line 133, can be simplified to "if not current_name.endswith('.csv'):"
- line 76 and 241, looks like display_outout should be a boolean True
- string literals "Test_Mode" and "DB_Mode" used many times in the code, replace them with constants will make possible future change much easier 
- line 342 and 348, please avoid writing multiple statements in one line

Everything else looks good to me!

breadsp committed:
Add files via upload

Applied all changes that Ming recommended
Updated the code to reflect the file-name changes per Anjan's validation rule sheet
